### ISSUE TITLE
Go to Line/Column - no line number in edit box

### ISSUE THREAD

**Issue by justas2481** [microsoft/vscode#298061]
Type: <b>Bug</b>

Open VSCode with NVDA screen reader enabled;
Open some text file;
Hit ctrl+g;
Use left/right arrow keys to see that the value is not there or not visible by NVDA.

VS Code version: Code 1.109.5 (072586267e68ece9a47aa43f8c108e0dcbf44622, 2026-02-19T19:43:32.382Z)
OS version: Windows_NT x64 10.0.26100
Modes:
Remote OS version: Linux x64 5.15.167.4-microsoft-standard-WSL2

**Comment by zewt** [comment]
Also having this issue, in 1.131.0 (e4c7e7b1d6).  I press ^G all the time to see which line I'm on, and now it's just saying "type a line number to go to".  Not related to screen readers for me, just ancient muscle memory ("need line number" = ^G, dating back at least to regular Visual Studio) that suddenly stopped working.
